rules/windows/credential_access_copy_ntds_sam_volshadowcp_cmdline.toml

```diff
@@ -38,11 +38,11 @@ query = '''
 process where host.os.type == "windows" and event.type == "start" and
   (
     (process.pe.original_file_name in ("Cmd.Exe", "PowerShell.EXE", "XCOPY.EXE") and
-       process.args : ("copy", "xcopy", "Copy-Item", "move", "cp", "mv")
+       process.args : ("*copy*", "*xcopy*", "*Copy-Item*", "*move*", "cp", "cp *", "mv", "mv *")
```

Comment:
Might be good to have a second thought with regards to the wildcards prior to the copy statements, might be resource intensive. Not sure how often a copy command is inserted in the middle of a command. Will leave that up to you, if you have good reasons to do so, LGTM!